src/containers/Settings/sagas.js

```diff
@@ -4,12 +4,33 @@ import { Toast } from 'components';
 import { actionTypes } from 'public-modules/Settings';
 
 const {
+  SAVE_SETTINGS_SUCCESS,
+  SAVE_SETTINGS_FAIL,
   SAVE_EMAIL_PREFERENCES_SUCCESS,
   SAVE_EMAIL_PREFERENCES_FAIL
 } = actionTypes;
 
-let lastToast;
-export function* sendEmailSettingsToast(action) {
+let lastProfileToast;
+export function* sendProfileUpdatedToast(action) {
+  let postedMessage = 'Profile saved successfully';
+  let toastType = Toast.TYPE.SUCCESS;
+
+  if (action.type === SAVE_EMAIL_PREFERENCES_FAIL) {
+    postedMessage = 'Something went wrong. Please try again.';
+    toastType = Toast.TYPE.ERROR;
+  }
+
+  const id = yield call(Toast, toastType, postedMessage, null);
+
+  if (lastProfileToast) {
+    callToast.dismiss(lastProfileToast);
```

Comment:
Does the order here affect visuals here? We're calling `yield call(Toast, toastType, postedMessage, null);` before we potentially call `callToast.dismiss(lastProfileToast);` - meaning the new toast would display before the old one was dismissed.

Reply:
These calls are so quick it will never be noticable.  But yeah - the correct ordering makes sense as you read the code